File: pgm/ABC_Visual_Basic/ABC_146_A.vb
' Imports System.Collections.Generic
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim S As String = Console.ReadLine()
        ' Dim ary As String() = {"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"}
        Console.WriteLine(7 - Array.IndexOf({"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"}, S))
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_146_A.cs
using System;
// using System.Collections.Generic;
// using System.Linq;

class Program
{
    static void Main()
    {
        string S = Console.ReadLine();
        Console.WriteLine(7 - Array.IndexOf(
            new string[] { "SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT" }, S));
        return;
    }
}